src/plugin.js

```diff
@@ -116,6 +141,13 @@ const initPlugin = function(player, options) {
         return;
       }
 
+      if (isIOS &&
+        (player.networkState() === 1 || player.networkState() === 2)) {
```

Comment:
I am a bit concerned that our regular timeout logic will no longer kick in on iOS. We should try to repro a timeout issue and see whether the network state is 1 or 2.